test-avocado/compiletest.sh

```diff
@@ -16,7 +19,7 @@ test -d $PACKAGE/$BUILD && /bin/rm -r $PACKAGE/$BUILD
 cd $PACKAGE
 mkdir -p $BUILD
 cd $BUILD
-../autogen.sh --prefix=/usr --enable-maintainer-mode --enable-debug
+../autogen.sh --prefix=/usr --libdir=/usr/lib$ARCH64 --enable-maintainer-mode --enable-debug
```

Comment:
We've discussed it last week. I would like to let this test not dependent on redhat (RPM)
I have also plan to have another compilation test based on  tools/make-rpms and probably further tools/make-deb or whatever. And In some cases use this test or test with  tools/make-rpms.
What do you thing? is it good idea, or it is better to remove that test and let ther only test with tools/make-*
    Regards
    Honza